### PR TITLE
Add repository configuration

### PR DESCRIPTION
**Pull request by mrunge**
This adds repository configuration for
- kibana
- elasticsearch
- grafana

### CODE REVIEW

**Comment by richm** [comment]
I thought we were using our own kibana?

**Comment by mrunge** [comment]
@richm current plan is to stick with upstream provided, i.e. with elasticsearch provided binaries.

File: roles/kibana/server/tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 ---
+- name: enable kibana repository
+  copy:
+      src: kibana.repo
+      dest: {{ yum_repos_d }}/kibana.repo
```
Comment (jbadiapa) [inline roles/kibana/server/tasks/main.yml]: dest:"{{ yum_repos_d }}/kibana.repo"

File: roles/elasticsearch/tasks/main.yml [hunk 2]
```diff
@@ -5,6 +5,12 @@
     state: present
   when: manage_packages|default(false)
 
+- name: enable elasticsearch repository
+  copy:
+      src: elasticsearch.repo
+      dest: {{ yum_repos_d }}/elasticsearch.repo
```
Comment (jbadiapa) [inline roles/elasticsearch/tasks/main.yml]: dest: "{{ yum_repos_d }}/elasticsearch.repo"

**Comment by jbadiapa** [comment]
I think at this point we should ensure that our playbooks work, so I think we need to start a new talk about the python bindings needed by ansible. 
I got this error msg:
.....Aborting, target uses selinux but python bindings (libselinux-python) aren't installed!....

**Comment by paramite** [comment]
@jbadiapa True, but that issue should be handled by another PR.

**Review by paramite — APPROVED** [review]

**Comment by paramite** [comment]
Ok, nobody is against this, I'm merging.

**Pull request merged**